src/wh_server_crypto.c

```diff
@@ -965,6 +1023,32 @@ int wh_Server_HandleCryptoRequest(whServerContext* server,
         ret = hsmCryptoCmac(server, (whPacket*)data, size, seq);
         break;
 #endif
+
+    case WC_ALGO_TYPE_HASH:
+        switch (packet->hashAnyReq.type) {
+            case WC_HASH_TYPE_SHA256:
+#ifdef DEBUG_CRYPTOCB_VERBOSE
+                printf("[server] SHA256 req recv. type:%u\n",
+                       packet->hashSha256Req.type);
+#endif
+                ret = hsmCryptoSha256(server, (whPacket*)data, size);
+#ifdef DEBUG_CRYPTOCB_VERBOSE
+                if (ret != 0) {
+                    printf("[server] SHA256 ret = %d\n", ret);
+                }
+#endif
+                break;
+
+            default:
+                ret = NOT_COMPILED_IN;
```

Comment:
Hrmmm... Had assumed we'd return CRYPTOCB_UNAVAILABLE, but this is more accurate since we are not necessarily in a cryptocb (client could just send the message).  Should we recheck how/if we want to handle this differently?

Reply:
I think it makes sense as written, right? We are essentially saying that the wolfHSM server doesn't support the client request for that algorithm, since support for that algorithm isn't compiled in.

Reply:
Concur.  I'm thinking we should extend wolfHSM error codes to support this status.